### ISSUE TITLE
Allow Builders to make Shrines (but only Priests/thralls can use them). 

### ISSUE THREAD

**Issue by CraftsmasterKubilai** [Tirocupidus/TheExiledRPOverhaul#122]
SUGGESTION: 

1. Make shrines (which are physical structures) buildable under the Builder profession, using current T1/T2/T3 rules.

2. Make is so that ONLY priests can actually CRAFT GOD ITEMS at those stations (canonically, this is the priest "sanctifying" the pile of bricks and stone, allowing the diety's power to manifest in the mundane, physical object). This would also foster the "interdependency" that TERPO aims to build by making Priests have to reach out to Builders to actually construct their shrines.. 

3. OPTIONAL (but recommended): Allow players to use thrall Priest NPC attached to the shrine to craft items ONE TIER LOWER THAN THE THRALL LEVEL (so actual player-controlled priests remain valuable). For example: a Level 1 NPC priests cannot craft anything independently for a PC (but do give a PC-controlled priest their crafting bonus), while any player can use a Level 2 priest NPC to craft *Level 1* god items, etc.

**Comment by gjpeebles** [comment]
2 already exists. As someone who likes realism, I like 1, but I'm not sure most people would be in favor of that, particularly with the currently higher cost of becoming a priest. Regarding 3, that also wouldn't mesh well with the high cost to become a priest, especially if T4 thralls can craft anything, negating the need for a priest at all (if builders can construct the altars).

**Comment by CraftsmasterKubilai** [comment]
Good observation re: high priest costs - maybe make the cost of lower-tier priests a bit lower/more in-line with other feats, reserving (as TERPO does now) the cost-scaling at the upper-end for T4? I've been felling for a while that priests "feel too expensive" for the benefit they give (compared to other crafter-types), so maybe this is a way to justify adjusting down those costs (since they'd need to be dependent on other professions - like everyone else - for some of their infrastructure)? As it stands now, only priests are basically "single purpose" characters, compared to others (who can get T4 in one feat +T2 in a second), meaning a much higher level of specialization/dedication.

**Comment by gjpeebles** [comment]
The main reason I was okay with their high cost before was because of the weapons. But without modding their stats back up, they don't make it worth it.

**Comment by CraftsmasterKubilai** [comment]
Yeah I can see that... under the current environment however, the Priest feat line seems a bit expensive for what you get IMHO. If you agree, this is one possible way to lower the feat cost to bring them more in-line with other professions while ALSO creating more opportunities for interdependency with others I was thinking. 

Also: that's a good point on god-specific weapons - I forgot about those, as well as god-specific harvesting tools. IMHO, since we're planning on letting thralls do a number of "QOL" things down the road like perform repairs, I'd suggest thrall priests be able to allow players to make the basic harvesting tool AS WELL AS the diety's "god food" (at the reduced level initially suggested in the OP, to add value to PC vs. thrall priests) but do NOT allow anyone but a true PC priest to make anything else (god weapons, avatar coins, etc.). 

Thanks for considering this!

**Comment by AlexandriaWolf** [comment]
I'd really like to see the priest line enhanced. It's far too expensive currently for what you get.
